Adds sharpness to swords

imtakingabreakdontatme opened the pull request:
It's a travesty that katanas currently do not work for dismemberment

phil235 committed:
* Admins can now bypass the vote creation cooldown.
* Messages in a mob's say_log var are now timestamped and include the coordinates of the mob at the time.
* Fix oil slime extract explosion reaction not being logged.
* Fixes lack of log for reagent explosion.
* Mob's attack_log now show when the mob was electrocuted. Doors get an electrification log when electrified (including via an EMP).
* Fixes gibself admin log showing null instead of the key.

phil235 committed:
Removes the voting cooldown fix and the say_log timestamp and coordinates.
Adds two files I forgot to add in the previous commit, somehow...

TechnoAlchemisto committed:
Gang tools no longer beep.

imtakingabreakdontatme committed:
Ship in a bottle

coiax committed:
Created simple maintloot counter

How much maint loot is there? This quickly hacked together python script
will tell you. Give it a map name, and wait.

coiax committed:
Counts duplicated tiles

coiax committed:
Using libraries is good yo

coiax committed:
Polite warnings, total printing

coiax committed:
Loot information for the log

Incoming5643 committed:
clients aren't datums so I'm forced into this slightly scummy situation. If at some point in the future byond makes clients datums this will need to be reverted.

coiax committed:
Maint table tweaks

Merged coloured gloves into one, added airlock painter, added secure
briefcases in preperation for the great mystery box ride.

coiax committed:
Renames ID cards in PDAs as well

nullbear committed:
adds QoL hotkey for pointing.

Ryll-Ryll committed:
Now the message for reloading admins comes after the reload

Commit:
Conversion to the gang now places your gang info in your memory/notes.

KorPhaeron committed:
Fixes for the ship in a bottle

KorPhaeron committed:
Fixes loot path

coiax committed:
pAI card plays boot_noise when personality loaded

- Added pai_boot.ogg sound file

https://www.freesound.org/people/jalastram/sounds/317638/

- Uses pai_boot.ogg noise when personality loads

coiax committed:
Adds additional holocarp colours

- Adds the Sand, Seashell, Coral, Salmon, Sunset and Riptide colours
  to the holocarp colour pool.

phil235 committed:
Dismemberment port from Hippie code, based on RemieRichard's work. Big thanks to RemieRichards and crystalwarrior.

coiax committed:
Added Robustin pantry

phil235 committed:
Some adjustments, following suggestions in the PR discussion.

phil235 committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into Dismemberment

Conflicts:
	code/datums/martial.dm
	code/game/gamemodes/changeling/powers/fleshmend.dm
	code/modules/mob/living/carbon/carbon.dm

phil235 committed:
always gotta fail the merged conflict fixing.

phil235 committed:
Fixes organ item path change on metastation map.

phil235 committed:
and birdstation...

phil235 committed:
and birdStation again...

phil235 committed:
Birdboat...

phil235 committed:
fixes the healthdoll.

phil235 committed:
Fixes the healthdoll not being updated right away after an amputation.

coiax committed:
Added maintenance fridge, mint, moved sink

coiax committed:
Removed step_[xy] for pixel_[xy]

imtakingabreakdontatme committed:
Adds sharpness to swords

xxalpha commented on code/game/objects/items/weapons/weaponry.dm:
```diff
@@ -68,6 +67,7 @@
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	block_chance = 50
+	sharpness = IS_SHARP
```
accurate please

Shapsy replied:
Have you ever cut a loaf of bread with a katana?

KorPhaeron replied:
Isn't accurate only used for surgery?

xxalpha replied:
Doesn't it influence meat cutting or am I wrong?

Also samurai surgeons.

ShadowDeath6 commented:
Glorious nippon steel

Incoming5643 commented:
Swords will fucking cut you wide open

Supermichael777 commented:
BANZIIIII

Bawhoppen commented:
@Incoming5643 Swords will fucking cut your hands off

octareenroon91 commented:
Pretty sure accurate matters for butchery

Shapsy commented:
It's only used for cutting bread as far as I can tell

ike709 commented:
https://www.youtube.com/watch?v=LlsgR4gVhEc

phil235 committed:
Fixes lizard body markings appearing behind the bodypart layer.
Fixes tail sprite when missing a leg and facing south.
Adds an admin option in human VV to remove a body part.
Fixes typos.
Fixes being able to toggle hardsuit and wintercoat helmet despite having no head.
Changes the dismemberment chance from explosion: bomb armor helps against it, and the maximum number of limbs that you can lose in one explosion is capped depending on the explosion severity.
Missing 1 or 2 legs makes you significantly slower. If without legs, missing an arm makes you slower. You can no longer move when missing all four limbs.
Bodyparts have now force 3 and throwforce 3
Tongues are now properly put in decapitated heads.
Shaking a legless person no longer gives weird results.

Zilenan91 commented:
Do broken bottles have sharpness

TechnoAlchemisto commented:
don't glass shards have is_sharp

Cheridan committed:
Merge pull request #17419 from phil235/Dismemberment

Dismemberment

tgstation-server committed:
Automatic changelog generation for PR #17419

ChangelingRain committed:
Merge pull request #17418 from coiax/holocarp-colours

Adds additional holocarp colours

ChangelingRain committed:
Merge pull request #17387 from coiax/hand-labeler-bug

Renames ID cards in PDAs as well when hand labeller suiciding

ChangelingRain committed:
Merge pull request #17416 from coiax/pai-boot-noise

pAI card plays a noise when personality is loaded

ChangelingRain committed:
Merge pull request #17379 from coiax/mainttally

Tools: Created simple maintloot counter

coiax committed:
Empty loot spawn back to 4

ChangelingRain committed:
Merge pull request #17415 from nullbear/crontroversial-fingerpointing-and-blame-pr

adds QoL hotkey for pointing.

tgstation-server committed:
Automatic changelog generation for PR #17415

ChangelingRain committed:
Merge pull request #17375 from KorPhaeron/bottle

Ship in a bottle

ChangelingRain committed:
Merge pull request #17382 from Incoming5643/badmins_doing_shit

Fixes var editing clients

ChangelingRain committed:
Merge pull request #17420 from coiax/robustin-pantry

New SW Maint Pantry (Box) (like #17283 but more mergable)

tgstation-server committed:
Automatic changelog generation for PR #17420

Commit:
Automatic changelog compile

KorPhaeron committed:
Merge pull request #17397 from Shapsy/reload_admins_reloaded

Reloading admins informs un-deadmin'd admins

Cheridan committed:
Merge pull request #17400 from bgobandit/whywouldyounotdothis

Conversion to the gang now places your gang info in your notes.

Cheridan committed:
Merge pull request #17331 from TechnoAlchemisto/gangfix

Gang tools no longer beep.

phil235 committed:
Merge branch 'master' of https://github.com/tgstation/-tg-station into AdminRequests1

Conflicts:
	code/modules/admin/verbs/randomverbs.dm

AnturK committed:
Merge pull request #17247 from phil235/AdminRequests1

Admin logging fixes and requests

AnturK committed:
Merge pull request #17385 from coiax/table-loot

Maint table tweaks

imtakingabreakdontatme committed:
Better dismemberment chances

imtakingabreakdontatme committed:
Merge branch 'sharp' of https://github.com/KorPhaeron/kortg into sharp

KorPhaeron commented:
Whoops
